db/column_family.cc

```diff
@@ -1102,10 +1102,10 @@ Status ColumnFamilyData::RangesOverlapWithMemtables(
       super_version->mem->NewRangeTombstoneIterator(read_opts, read_seq);
   range_del_agg.AddTombstones(
       std::unique_ptr<FragmentedRangeTombstoneIterator>(active_range_del_iter));
-  super_version->imm->AddRangeTombstoneIterators(read_opts, nullptr /* arena */,
-                                                 &range_del_agg);
-
   Status status;
+  status = super_version->imm->AddRangeTombstoneIterators(
```

Comment:
Since AddRangeTombstoneIterators always returns OK, maybe just change it to return a void instead?

Reply:
Yes, I wasn't sure if I should  change the API's (return type) for this change.

Reply:
Can you add a comment or PR that says the method always returns OK so it can be addressed sometime later?